lexicon/tests/providers/test_dreamhost.py

```diff
@@ -0,0 +1,27 @@
+"""Integration tests for Linode"""
+from unittest import TestCase
+
+import pytest
+from lexicon.tests.providers.integration_tests import IntegrationTests
+
+
+# Hook into testing framework by inheriting unittest.TestCase and reuse
+# the tests which *each and every* implementation of the interface must
+# pass, by inheritance from integration_tests.IntegrationTests
+class DreamhostProviderTests(TestCase, IntegrationTests):
+    """TestCase for Linode"""
```

Comment:
You should update you comment to put Dreamhost instead of Linode.

Reply:
I'll change to Dreamhost.